frontend/src/concepts/pipelines/content/createRun/utils.ts

```diff
@@ -45,6 +46,7 @@ export const isFilledRunFormData = (formData: RunFormData): formData is SafeRunF
 
   return (
     !!formData.nameDesc.name &&
+    formData.nameDesc.name.length <= RUN_NAME_CHARACTER_LIMIT &&
```

Comment:
Yes, we still allow users to create runs with duplicate names. That's what the text the JIRA screenshot had when the icon is of type "info", so that's a question for Yan and/or Katie.

Reply:
@yannnz @kaedward ^ What do you think? We can also merge this PR and create a ticket for this later.

Reply:
I think it makes sense to info the user about the duplicated name. IMO, if I am the user and uses the same names for different runs, it will confuse me later to find the correct run I am looking for. I will try to avoid using duplicated name as a user for better filtering and management. I don't understand why technically the Platform allows so, but if it is how it works nowadays, I would think it is good to use info icon to warn the user.

Reply:
Thanks @yannnz. Unless @kaedward has a suggestion for different wording here, I think we roll with this for now at least.